packages/storage-evm/scripts/base-mainnet-contracts-scripts/test-executor-mint.ts

```
@@ -0,0 +1,691 @@
+
+      if (proposalData.expired) {
+        console.log('   ⏰ Proposal has expired without execution');
+      }
+    }
+  } catch (error) {
+    console.error('❌ Error during test execution:', error);
+
+    if (error.message && error.message.includes('f')) {
+      console.log('\n🔍 This might be the "f" error you mentioned earlier.');
+      console.log('   Possible causes:');
+      console.log('   1. Token contract executor address mismatch');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)